src/components/CookieBanner/CookieBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import { CgClose } from "react-icons/cg";
+import CookieIcon from '../../img/cookieIcon.png'
+import VhiLogo from '../../img/cookieVhiLogo.png'
+
+export default function CookieBanner() {
+
+    const [isVisible, setIsVisible] = useState(false);
+
+    function ShowCookieBanner() {
+        setIsVisible(true);
+    }
+
+    function hideCookieBanner() {
+        setIsVisible(false);
+    }
+
+    return (
+        <div className='util-cookie-banner'>
+            <div className={`util-cookie-banner-container util-cookie-banner-container--${isVisible ? "show" : "hide"}`}>
```

Comment:
all lower case for the class name plz (--visible or --invisible)

Reply:
Not sure the issue here? The class name will end with --show or --hide. Are you saying --visible or --invisible would be a better description?